openai/api_requestor.py

```diff
@@ -392,7 +393,7 @@ def handle_error_response(self, rbody, rcode, resp, rheaders, stream_error=False
             )
 
     def request_headers(
```

Comment:
Nit: can you check if the call in line 490 needs to get updated based on this?

Reply:
Yep, thank you for the review. It's indeed a bug that I overlooked. I've just fixed it in [dc8feeb](https://github.com/openai/openai-python/pull/345/commits/dc8feeb933f97e72a0a266a187d9c28ada45d8de)